sandbox/src/mocks/cancelAppointmentActionLater/responses/MinimalExampleIBS.json

```diff
@@ -42,6 +42,19 @@
 				"system": "https://fhir.nhs.uk/Id/ods-organization-code",
 				"value": "R69"
 			}
+		},
+		{
+			"url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ReferralState-1",
+			"extension": [
+				{
+					"url": "state",
+					"valueCoding": {
+						"system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralState-1",
+						"code": "NOT_BOOKED",
```

Comment:
I think the referral would be in a `BOOKED` state in this scenario

Reply:
The contained appointment is deferral with Proposed State. So Should it be NOT_BOOEED rather than BOOKED?